flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -1144,6 +1186,9 @@ public void run() {
 
 				checkpointMetrics.setAsyncDurationMillis(asyncDurationMillis);
 
+				bufferPersistingFuture.get();
+
+				LOG.debug("{}: Finished checkpoint {}.", owner.getName(), checkpointMetaData.getCheckpointId());
```

Comment:
I guess this log is not necessary. Otherwise the whole log stacks would be following:

`Finished synchronous checkpoints.......`
`Finished checkpoint......`
`finished synchronous part of checkpoint.....`

It might confuse users.